packages/botonic-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@botonic/react",
-  "version": "0.10.1",
+  "version": "0.11.0-alpha.0",
```

Comment:
@dpinol we already published a version 0.11.0-alpha.0 https://www.npmjs.com/package/@botonic/react/v/0.11.0-alpha.0 which was needed by Zurich. Maybe rename it as 0.11.0-alpha.1 better.

Reply:
good point, actually the last commit of the PR already upgrades to alpha.1
It's nice that npm (I guess when publishing) automatically upgrades from alpha/beta.X to alpha/beta.X+1